ts/components/staking/layout/staking_page_layout.tsx

```diff
@@ -29,6 +31,7 @@ export const StakingPageLayout: React.FC<IStakingPageLayoutProps> = props => {
             />
 
             {props.children}
+            <ConnectWalletDialog />
```

Comment:
Not sure if this is the best place to put the dialog component but here it is ensured to always be mounted on a page where a "Connect Wallet" button can exist. Wdyt?